src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -1351,6 +1350,16 @@ Defines the DOCKER_HOST environment variable. The DOCKER_HOST variable defines t
 That property is useful when accessing a Docker daemon which runs inside a container or on a remote machine.
 Further information about the DOCKER_HOST property can be found http://docs.docker.com/v1.8/reference/commandline/cli/[in the official Docker documentation^].
 
+=== Perl scripts execution (Perl task)
+Perl should be installed on your machine. By default the perl engine is installed in Linux and Mac Os. Please install the perl engine on Windows. +
+The execution mechanism of perl task is based on running perl file on your machine.
```

Comment:
The execution mechanism of perl tasks is based on running perl files on your machine.